### PR TITLE
Use a version wildcard for MEL dependency

### PR DESCRIPTION
**Pull request by simenhagelid**
Added floating version for PackageReference  Microsoft.Extensions.Logging to 2.*

### CODE REVIEW

**Comment by nblumhardt** [comment]
Thanks! Let's give this a try 👍 

I've re-targeted it to `dev` - will squash and merge to try cleaning up the history a bit :-)

File: src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj [hunk 1]
```diff
@@ -31,7 +31,7 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net461' or '$(TargetFramework)' == 'netstandard2.0' ">
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.*" />
```
Comment (SimonCropp) [inline src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj]: note that this wont effect the resulting package. on each build the resultant nuget will have the current release version of Microsoft.Extensions.Logging hard coded as a package dependency

Comment (nblumhardt) [reply src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj]: Argh, thanks Simon - lost track of this particular footgun :-)

Comment (SimonCropp) [reply src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj]: I only remember due to my missing foot

Comment (nblumhardt) [reply src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj]: 😂

**Pull request merged**